esp-config/src/generate.rs

```diff
@@ -15,27 +15,37 @@ pub struct ParseError(String);
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub enum Value {
-    Number(usize),
+    SignedInteger(isize),
```

Comment:
Do we expect something like `-0xAA` to work? Right now it's not supported but maybe it should be?

Reply:
Let's not get down this rabbit hole, otherwise we'll end up with a complete evaluation engine. If someone uses a bit pattern (binary or hexadecimal), let's not allow modifiers on that.